workflows/config/mcmc_config_mark00.json

```diff
@@ -0,0 +1,12 @@
+{
+    "move_probs": {
+        "prune_and_reattach": 0.1,
+        "swap_node_labels": 0.65,
+        "swap_subtrees": 0.25
+    },
+    "fnr": 0.097,
+    "fpr": 1.24e-06,
+    "num_samples": 100,
+    "burn_in": 10,
+    "thinning": 2
+}
```

Comment:
I decided to use a config file for `run_mcmc.py`to pass all these arguments as I don't expect them to change that much and it would be cumbersome to pass so many arguments each time. What do you think?

Reply:
There are two ways how this is often done:

1. Pydantic's `BaseModel` can have default values and have hierarchical configuration.
2. [Hydra](https://hydra.cc/docs/intro/) is Facebook AI Research's solution of maintaining hierarchical configurations.

We should consider perhaps using one of these, but it's definitely out of scope for this PR :slightly_smiling_face:

Reply:
Yes, please teach me. Would like to learn the good way. Perhaps something for the next Sprint. Will make an issue, so we don't forget. See #35.